server/src/main/java/com/objectcomputing/checkins/services/memberprofile/MemberProfileController.java

```diff
@@ -25,7 +24,7 @@ public MemberProfileController(MemberProfileRepository memberProfileRepository){
         this.memberProfileRepository = memberProfileRepository;
     }
 
-    @Get("/{?name,role,pdlId}")
+    @Get(uri = "/{?name,role,pdlId}", produces = MediaType.APPLICATION_JSON)
```

Comment:
You should be able to avoid repeating this on each method by annotating the class with `@Produces`, I believe:
https://docs.micronaut.io/latest/api/io/micronaut/http/annotation/Produces.html